scripts/install_onnx_gpu_for_amd.sh

```diff
@@ -7,7 +7,7 @@ if ! [[ "$TARGETPLATFORM" ]]; then
 fi
 
 if [[ "$TARGETPLATFORM" != "linux/arm64" ]]; then
-  pip3 --no-cache-dir install --upgrade onnxruntime-gpu
+  pip3 --no-cache-dir install --upgrade onnxruntime-gpu==1.12.1
 else
   pip3 --no-cache-dir install --upgrade onnxruntime
```

Comment:
this should probably be the same as in requirements.txt

Reply:
I tried that, in another branch, but it didn't help: https://github.com/marqo-ai/marqo/blob/pinning_reqs_protobuf/scripts/install_onnx_gpu_for_amd.sh